src/domain/relay/errors/invalid-multisend.error.ts

```diff
@@ -0,0 +1,7 @@
+export class InvalidMultiSendError extends Error {
+  constructor() {
+    super(
+      'Invalid `multiSend` call. The batch is not all `execTransaction` calls to same address.',
```

Comment:
nitpick: I'd remove the ` as I assume this is mostly for rendering in markdown.

Reply:
Removed in 4a9d3787f8f6f122c9eb31a6e5e4cb00acc1c6e4.